astropy/stats/tests/test_biweight.py

```diff
@@ -558,15 +558,6 @@ def test_biweight_midcorrelation_inputs():
         biweight_midcorrelation(a2, a3)
 
 
-def test_biweight_32bit_runtime_warnings():
-    """Regression test for #6905."""
```

Comment:
xref #6905

Reply:
This should stay, `np.float32` arrays can be used on 64bit systems.